src/test/scala/NumberFormatTest.scala

```diff
@@ -137,8 +150,9 @@ class NumberFormatTest extends FunSuite {
   test(
     "should throw NumberFormatError when the pattern provided is not valid"
   ) {
-    intercept[NumberFormatError] {
-      NumberFormat(Some("0.#00#"))
-    }
+    // Update: Not validating numeric data-types with formats
+//    intercept[NumberFormatError] {
+//      NumberFormat(Some("0.#00#"))
+//    }
```

Comment:
Might be safter to just comment out all of these tests so we don't see them and think they're running and passing.